### ISSUE TITLE
Autoplay doesn't hide Play button in Listen mode

### ISSUE THREAD

**Issue by Discookie** [omarroth/invidious#224]
![issue](https://user-images.githubusercontent.com/16914176/48164090-d1317b00-e2e1-11e8-8546-0958b0800b64.png)

Note the sound icon in the header. I get this every 5 or so reloads, even if I'm not in a playlist.
I'm on Waterfox latest, Windows x64.

(I've been struggling to get Autoplay to work at all on Chrome, even when setting #autoplay-policy to require no gestures)

**Comment by omarroth** [comment]
I'm having trouble getting it to work on Chrome as well, but 7a6d4e6 appears to fix the issue in Firefox. My guess is that video ads etc have led to much stricter controls on autoplay, and other people have had issues with this, and from the [video.js blog itself](https://blog.videojs.com/autoplay-best-practices-with-video-js/):
>## tl;dr
> - Never assume autoplay will work.
> - Using the muted attribute/option will improve the chances that autoplay will succeed.
> - Prefer programmatic autoplay via the player.play() method, avoiding the autoplay attribute/option.

**Comment by Discookie** [comment]
Issue is still present on Waterfox:
![image](https://user-images.githubusercontent.com/16914176/48653416-5238f280-ea05-11e8-8504-b1a9d382c5d0.png)


The play button's gone though! :P - but the controls still don't show.

**Comment by omarroth** [comment]
https://invidio.us/watch?v=03hOLCfgDps&list=RD03hOLCfgDps&autoplay=1&listen=1 works as expected for me on Waterfox. You might try refreshing your cache or checking devtools for any errors.

**Comment by Discookie** [comment]
Nope, I still get it on every second refresh.  
I can reliably reproduce this behavior by refreshing while the previous track is playing.  
I also get no warnings at all, other than the `videojs.plugin()` and the `frame-src` deprecation notices.

It's as if some trigger that signals the 'Video loaded!' state is never fired:
* On load, the video plays with no controls.
* On the first click it pauses, still no controls.
* From the second click on wards, the controls are visible and it works as expected.

**Comment by omarroth** [comment]
Should be fixed with 7ab9d741bfc36e19c703ea208a4d39b3a3c78587.

**Comment by Discookie** [comment]
Still getting it.

After experimenting with it for a bit (with HTTP caching disabled), it looks like the audio starts playing before the player finishes loading.

* The play button shows for a split second.
* It disappears, and the music starts playing, with a fully black videobox.
* The thumbnail/background loads, and takes up the full screen, but the controls don't show until I click it, as in the above comment.

**Comment by Discookie** [comment]
It's fixed for cache-clearing refreshes and video switches.
Apparently it's still broken on regular refreshes.

**Comment by omarroth** [comment]
If you're still encountering this, would you mind including the exact browser version you're running? I'm having trouble reproducing in Chrome, Waterfox, Firefox, or Palemoon.

**Comment by Discookie** [comment]
I can't seem to reproduce it anymore, although I have no idea when it fixed itself.
I'll reopen the issue if I manage to reproduce it on a different machine.
